doc/source/index.rst

```diff
@@ -1,91 +1,56 @@
-Welcome to SNMF's Documentation!
```

Comment:
here also, just move the old file over to the new proejct so wee keep everything.  I don't think there is anything we need from the new template.